katharsis-jpa/src/test/java/io/katharsis/jpa/query/JoinedTableInheritanceQueryTestBase.java

```diff
@@ -0,0 +1,22 @@
+package io.katharsis.jpa.query;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import io.katharsis.jpa.model.JoinedTableBaseEntity;
+import io.katharsis.jpa.model.JoinedTableChildEntity;
+
+public abstract class JoinedTableInheritanceQueryTestBase
+		extends AbstractInheritanceTest<JoinedTableBaseEntity, JoinedTableChildEntity> {
+
+	public JoinedTableInheritanceQueryTestBase() {
+		super(JoinedTableBaseEntity.class, JoinedTableChildEntity.class);
+	}
+
+	@Override
+	@Test
+	@Ignore
+	public void testOrderBySubtypeAttribute() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)